tests/test_buffer_dynamic.py

```diff
@@ -636,3 +636,40 @@ def test_autoNegPort(self, dvs, testlog):
 
         # Shutdown interface
         dvs.runcmd('config interface shutdown Ethernet0')
+
+    def test_removeBufferPool(self, dvs, testlog):
+        self.setup_db(dvs)
```

Comment:
please put this in a try-finally block so that the cleanup always happens

Reply:
Fixed